tests/validation/cattlevalidationtest/core/test_native_docker.py

```diff
@@ -112,7 +101,7 @@ def check():
     r_containers = admin_client.list_container(name=name)
     assert len(r_containers) == 1
     c = r_containers[0]
-    c = admin_client.wait_success(c)
+    c = admin_client.wait_success(c, timeout=180)
```

Comment:
Increased timeout to account for network-agent container taking time to download and install on fresh environment.